qiskit/qpy/type_keys.py

```diff
@@ -303,6 +309,11 @@ class ScheduleOperand(TypeKeyBase):
     # Data format of these object is somewhat opaque and not defiend well.
     # It's rarely used in the Qiskit experiements. Of course these can be added later.
 
+    # A temp hack. If pulse instruction operands involve a string,
```

Comment:
Heh, well it's not going to be so temporary if we push this out as version 7 we'll need to support `o` as a type key for string forever as we'll always need to be able to load version 7. But it's also not the end of the world from my perspective, we're just using letters because it's easier to reason about compared to `0x6F`. So the fact that we're already using `s` in this context it doesn't seem to be such a big deal we have to use a different character for string here. I think we already have a conflict like this for something on the circuit side.

Reply:
Fair enough. I updated the comments.